src/renderer/components/App/AppIntro/AppIntro.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+  <div
+    class="pt-10 pb-6 px-10 flex flex-col rounded-lg w-full h-full pin-t pin-l fixed animated fadeIn lg:overflow-y-scroll"
+  >
+    <div
+      class="flex w-full h-full pin-t pin-l"
+    >
+      <AppIntroScreen
+        v-if="step === 0"
+        :show-back="false"
+        :show-next="false"
+        :show-skip="false"
+        :show-logo="false"
+        :image="stepImage"
+        class="AppIntro__1"
+      >
+        <div
+          slot="content"
+          class="w-2/3 font-medium"
+        >
+          <i18n
+            path="INTRODUCTION.WELCOME.TITLE"
+            tag="div"
+            class="font-bold mb-4"
+          >
+            <p
+              place="APP"
+              class="font-black text-3xl"
+            >
+              {{ $t('COMMON.APP_NAME_SHORT') }}
+            </p>
+          </i18n>
+          <p>
+            {{ $t('INTRODUCTION.WELCOME.SAFETY_MESSAGE') }}
+          </p>
+          <p class="mt-2">
+            {{ $t('INTRODUCTION.WELCOME.FUNDS_WARNING') }}
+          </p>
+
+          <div
+            class="flex flex-row align-center justify-center mt-8"
+          >
+            <MenuDropdown
+              :items="languages"
+              :value="language"
+              :position="['-50%', '0%']"
```

Comment:
This positioning is awkward because it is misaligned to the input. I think you making the same adjustments made to the ProfileEdition will look better.

Reply:
Thanks, I didn't realize how awkward was. I've also improved the style of the items.